chocs/http_response.py

```diff
@@ -39,6 +40,9 @@ def writable(self):
     def close(self):
         self.body.close()
 
+    def set_cookie(self, cookie: Cookie):
```

Comment:
Can we have property setter here, I think it will be more pythonic way in this scenario.

```python
@property
def cookies() -> Cookie:
  ...
@cookies.setter()
def cookies(value: Cookie) -> None:
  ...
```

Reply:
How would `@property` work in this case, would have to store the cookies separately from other headers so they can be returned without other headers. That might be useful, but I want to be sure that's what you are suggesting here.

Reply:
We could store a `list` of cookies then add them to the `Headers` when we call `headers()` property

Reply:
Ah I see so this is add_cookie actually, maybe we should have a cookie aggregate in this scenario so end interface might look like this;
`Request.cookies.add(Cookie())` what do you think?